src/dotnet/commands/dotnet-new/FSharp_Web/$projectName$.fsproj

```diff
@@ -10,20 +10,10 @@
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wp8+wpa81;</PackageTargetFallback>
   </PropertyGroup>
 
-  <PropertyGroup>
-    <GlobalExclude>$(GlobalExclude);bin\**;obj\**;node_modules\**;jspm_packages\**;bower_components\**;**\*.user;**\*.*proj</GlobalExclude>
-  </PropertyGroup>
-
   <ItemGroup>
-    <None Include="**\*" />
     <Compile Include="Controllers\*.fs" />
     <Compile Include="Startup.fs" />
```

Comment:
Should `<Compile Remove="**\*.cs" />` be added to account for them being in the WebSDK props, or is it OK to leave them?